src/main/java/com/config/TWConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package com.config;
+
+import io.dropwizard.Configuration;
+import twitter4j.conf.ConfigurationBuilder;
+
+public class TWConfiguration extends Configuration {
+
+    public static ConfigurationBuilder configurationBuilder()
+    {
+        ConfigurationBuilder cb = new ConfigurationBuilder();
```

Comment:
rename cb object name

Reply:
did u get a chance to change object name here, follow sensible variable naming conventions

Reply:
changed